[Bug] - S3 Logger Fails if context is nil

**Describe the bug**
Application panics when a context string is not provided to the logger and provides the following error:
```
panic: interface conversion: interface {} is nil, not string

goroutine 75514 [running]:
github.com/versity/versitygw/s3log.(*FileLogger).Log(0xc00041c150, 0xc000d26308, {0x1856800, 0xc000314450}, {0x0?, 0x0, 0x7fd63493d4a0?}, {{0x0, 0x0}, 0x0, ...})
        /app/s3log/file.go:125 +0x99a
github.com/versity/versitygw/s3api/controllers.SendResponse(0xc000d26308, {0x1856800, 0xc000314450}, 0xc0011fbad8)
        /app/s3api/controllers/base.go:3907 +0xe6
github.com/versity/versitygw/s3api.New.DecodeURL.func2(0xc000d26308)
        /app/s3api/middlewares/url-decoder.go:31 +0x130
github.com/gofiber/fiber/v2.(*App).next(0xc0003def08, 0xc000d26308)
        /go/pkg/mod/github.com/gofiber/fiber/v2@v2.52.8/router.go:143 +0x1a7
github.com/gofiber/fiber/v2.(*App).handler(0xc0003def08, 0x4d2673?)
        /go/pkg/mod/github.com/gofiber/fiber/v2@v2.52.8/router.go:170 +0x69
github.com/valyala/fasthttp.(*Server).serveConn(0xc00015ab48, {0x186ead0, 0xc000698018})
        /go/pkg/mod/github.com/valyala/fasthttp@v1.62.0/server.go:2455 +0x11cf
github.com/valyala/fasthttp.(*workerPool).workerFunc(0xc0004202d0, 0xc001811600)
        /go/pkg/mod/github.com/valyala/fasthttp@v1.62.0/workerpool.go:225 +0x92
github.com/valyala/fasthttp.(*workerPool).getCh.func1()
        /go/pkg/mod/github.com/valyala/fasthttp@v1.62.0/workerpool.go:197 +0x32
created by github.com/valyala/fasthttp.(*workerPool).getCh in goroutine 106
        /go/pkg/mod/github.com/valyala/fasthttp@v1.62.0/workerpool.go:196 +0x194
```

**To Reproduce**
Unknown steps to reproduce, happens regularly during security scans.

**Expected behavior**
The app should handle having a nil context in the logger.

**Server Version**
```
Version  : v1.0.16
Build    : b2516e415359971328a11676c7e6fd15651331ed
BuildTime: 2025-07-18T15:21:42Z
Linux cb4f04cdd263 5.14.0-570.28.1.el9_6.x86_64 #1 SMP PREEMPT_DYNAMIC Wed Jul 16 02:51:06 EDT 2025 x86_64 Linux
```

App is in the official v1.0.16 container with the digest `174d24e7f0de`

[174d24e7f0de](https://hub.docker.com/layers/versity/versitygw/v1.0.16/images/sha256-174d24e7f0de4d90e5898f09e35f88ae7735ac6504e511ecc7f2b3a5eca3934e)

**Additional context**
Client is a proprietary storage client made with the AWS S3 SDK. From the access logs it identifies as aws-sdk-cpp/1.7.211

--- a/s3log/file.go
+++ b/s3log/file.go
@@ -101,6 +101,11 @@ func (f *FileLogger) Log(ctx *fiber.Ctx, err error, body []byte, meta LogMeta) {
 		access = acct.Access
 	}
 
+	region, ok := utils.ContextKeyRegion.Get(ctx).(string)
+	if ok {
+		lf.HostHeader = fmt.Sprintf("s3.%v.amazonaws.com", region)
+	}
+
 	lf.BucketOwner = meta.BucketOwner
 	lf.Bucket = bucket
 	lf.Time = time.Now()
@@ -122,7 +127,6 @@ func (f *FileLogger) Log(ctx *fiber.Ctx, err error, body []byte, meta LogMeta) {
 	lf.HostID = ctx.Get("X-Amz-Id-2")
 	lf.SignatureVersion = "SigV4"
 	lf.AuthenticationType = "AuthHeader"
-	lf.HostHeader = fmt.Sprintf("s3.%v.amazonaws.com", utils.ContextKeyRegion.Get(ctx).(string))
 	lf.AccessPointARN = fmt.Sprintf("arn:aws:s3:::%v", strings.Join(path, "/"))
 	lf.AclRequired = "Yes"
 

